src/core/connection.c

```diff
@@ -304,6 +304,9 @@ QuicConnFree(
                 AllStreamsLink);
         QUIC_DBG_ASSERTMSG(Stream != NULL, "Stream was leaked!");
     }
+    for (uint32_t i = 0; i < ARRAYSIZE(Connection->Packets); i++) {
+        QUIC_FRE_ASSERT(Connection->Packets[i] == NULL);
```

Comment:
Actually, use a debug assert. We don't want a leak to actually bugcheck a production system.

Reply:
Its already in a #Debug